### ISSUE TITLE
[Bug]: `@guardian/consent-management-platform` peer dependency mismatch

### ISSUE THREAD

**Issue by cemms1** [guardian/dotcom-rendering#10206]
### What happened?

There is a peer dependency mismatch in DCR for `@guardian/consent-management-platform` 

```
@guardian/consent-management-platform 13.7.1
│ └── ✕ unmet peer @guardian/libs@^15.0.0: found 16.0.0
```

> [!TIP]
> You can check this yourself by running `pnpm install --resolution-only`

@guardian/transparency-consent please could you update this package to use versions that are compatible with this repo?

### What is the impact?

This will soon become a blocker as we want to enforce met peer dependencies in this repo

**Comment by JamieB-gu** [comment]
@alinaboghiu to check who is picking this up.

**Comment by sookburt** [comment]
[PR](https://github.com/guardian/consent-management-platform/pull/908) to bump guardian/libs peer dependency to v16 merged and [release PR](https://github.com/guardian/consent-management-platform/pull/911) ready to go once we have bumped our dependencies for a previous release with a deadline of today.  Planned for next release.

**Comment by sookburt** [comment]
> > [release PR](https://github.com/guardian/consent-management-platform/pull/911) ready to go once we have bumped our dependencies for a previous release with a deadline of today. Planned for next release.
> 
> @sookburt I've created and merged this PR #10546. Please let me know if that's ok. We can revert if not, but keen to complete this work

I believe this is fine thanks.
